tests/integration/test_manifest_generation.py

```diff
@@ -498,6 +498,670 @@ def test_single_manifest_generation_google_sheet(
+        - The first row is locked on scroll
+        - Each cell A-F in the first row has a comment "TBD"
+        - Cell G in the first row does not have a comment
+        - The cell C2 in "Sheet1" has a dropdown list with values from "Sheet2!C2:C5"
+        - The cell E2 in "Sheet1" has a dropdown list with values from "Sheet2!E2:E5"
+        - The fill colors of the first row cells are as expected
+        - The workbook contains two sheets: "Sheet1" and "Sheet2"
+        - "Sheet2" is hidden
+        - The values in "Sheet1" are as expected
+        - The values in "Sheet2" are as expected
+
+        Manual verification steps:
+        - When File Format = "BAM", [Genome Build] is Light Blue (Required)
+        - When File Format = "CRAM", [Genome Build, Genome FASTA] is Light Blue (Required)
+        - When File Format = "FASTQ", [Genome Build] is White (Optional)
+        """
+        url = "http://localhost:3001/v1/manifest/generate"
+        params = {
+            "schema_url": "https://raw.githubusercontent.com/Sage-Bionetworks/schematic/develop/tests/data/example.model.jsonld",
+            "title": "Example",
+            "data_type": "BulkRNA-seqAssay",
+            "use_annotations": "false",
+            "dataset_id": "syn63561056",
+            "asset_view": "syn63561086",
+            "output_format": "google_sheet",
+            "strict_validation": "true",
+            "data_model_labels": "class_label",
+        }
+        headers = {
+            "accept": "application/json"
+            # TODO: Include an authorization header
+        }
+        # WHEN we make a request to the Schematic API
+        response = requests.get(url, headers=headers, params=params)
+
+        # THEN we expect a successful response
+        assert response.status_code == 200
+
+        # Load the Google Sheets URL from the response
+        response_content = response.json()
+        assert len(response_content) == 1, "Expected a single URL in the response"
+        google_sheet_url = response_content[0]
+        assert (
+            google_sheet_url is not None
+        ), "No Google Sheets URL found in the response"
+
+        # Convert the Google Sheets URL to an export URL for Excel format
+        export_url = f"{google_sheet_url}/export?format=xlsx"
+        response = requests.get(export_url)
+        assert response.status_code == 200
+        content = BytesIO(response.content)
+        workbook = load_workbook(content)
+        sheet1 = workbook["Sheet1"]
+        sheet2 = workbook["Sheet2"]
+
+        # AND the column headers in "Sheet1" are as expected
+        assert sheet1["A1"].value == "Filename"
+        assert sheet1["B1"].value == "Sample ID"
+        assert sheet1["C1"].value == "File Format"
+        assert sheet1["D1"].value == "Component"
+        assert sheet1["E1"].value == "Genome Build"
+        assert sheet1["F1"].value == "Genome FASTA"
+        assert sheet1["G1"].value == "entityId"
```

Comment:
One of the things I had run into was that the column ordering can change. You have that `track column positions` code below, but these assertions may fail one day. From what @andrewelamb had mentioned, let's operate under the assumption that column order may change, and worry about ensuring a consistent order later on.